paasta_tools/tron_tools.py

```diff
@@ -270,6 +271,20 @@ def get_cleanup_action(self, default_paasta_cluster):
         action_dict['name'] = 'cleanup'
         return self._get_action_config(action_dict, default_paasta_cluster)
 
+    def check_monitoring(self) -> Tuple[bool, List[str]]:
+        msgs: List[str] = []
+        monitoring = self.get_monitoring()
+        checks_passed = True
+        if monitoring is not None:
+            team_name = monitoring.get('team', None)
+            if team_name is None:
+                checks_passed = False
+                msgs.append('Team name is required for monitoring')
+            elif team_name not in list_teams():
```

Comment:
In our typical team list, is 'noop' a valid team? Or does it need to be a special case (as it is in the pre-receive hook)? 

Also, maybe we should provide suggestions if there's a valid team name that is similar.

Reply:
noop is a valid team name. suggestion is added.